keras_imaging/preprocessing/readers.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
```

Comment:
I don’t understand the purpose of this module. Would you mind explaining your intention?

From my perspective, we’re requiring scikit-image, so we can expect users to have scikit-image installed. Moreover, the default reader for scikit-image is [imageio](https://imageio.github.io/) and that package manages libtiff and pillow use. Nonetheless, I understand the desire for specifying a particular reader.

What about piggybacking scikit-image’s [plug-in interface](http://scikit-image.org/docs/dev/api/skimage.io.html)? The constructor could have a `plugin` parameter that is passed to `skimage.io.imread`.